src/commands/moderation/clearwarning.ts

```diff
@@ -0,0 +1,36 @@
+import { CommandCategories, CommandDefinition, createErrorEmbed } from '../index';
+import { createEmbed } from '../../lib/embed';
+import Warning from '../../schemas/warning';
+
+export const clearwarning: CommandDefinition = {
+    names: ['clearwarning', 'clearwarn'],
```

Comment:
Honestly I would just rename this entire thing to 'removewarning' (alias rmwarning?). Clear should be used for more than one operations (e.g. multiple warnings)

(repost because I thought I was doing a review comment :leBriar:)